src/main/utils/resource.ts

```diff
@@ -60,9 +60,9 @@ export class Resource {
      * if the given file is a binary, replace and copy it to the target path
      * @param tempPath - the file to copy into target
      */
-    public copyToTarget(tempPath: string) {
+    public async copyToTarget(tempPath: string) {
```

Comment:
This function is async now, so to be more consistent, let's replace all other sync functions with async ones, such as copyFileSync -> copyFileSync